src/zenml/integrations/graphviz/__init__.py

```diff
@@ -22,7 +22,6 @@ class GraphvizIntegration(Integration):
 
     NAME = GRAPHVIZ
     REQUIREMENTS = ["graphviz>=0.17"]
-    SYSTEM_REQUIREMENTS = {"graphviz": "dot"}
```

Comment:
graphviz seems to be available via apt, how come we can't use the new `APT_PACKAGES ` here?

Reply:
The `APT_PACKAGES` are essentially just used when a stack component of an integration is in the active stack when running a pipeline, it wouldn't really have any impact to add it for graphviz as that integration contains no stack component and also serves no real purpose while running pipeline. Would you still add it in this case for completeness?

Reply:
Ah I see. No, then it's fine as it is 👍